tests/integration/loki-tester/.flake8

```diff
@@ -0,0 +1,9 @@
+[flake8]
```

Comment:
Even for tester charms I would use operator-template rather than charmcraft init.